src/test/java/nl/tudelft/scrumbledore/BubbleTest.java

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.scrumbledore;
+
+//import static org.junit.Assert.*;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+//import org.junit.Test;
+
+/**
+ * Testing the Bubble class.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class BubbleTest extends ProjectileTest{
+
+  /**
+   * Setting up test properties.
+   * 
+   * @throws Exception
+   */
+  @BeforeClass
+  public static void setUpBeforeClass() throws Exception {
+  }
+
+  /**
+   * Deleting test properties after testing.
+   * 
+   * @throws Exception
+   */
+  @AfterClass
+  public static void tearDownAfterClass() throws Exception {
+  }
+
+  @Override
+  public LevelElement make(Vector position, Vector size) {
```

Comment:
1. Are these kind of methods not redundant?
It takes the same amount of code to make a new instance of a Bubble instead of calling this method.
2. Do you want to return a LevelElement or a Bubble? You are creating a Bubble and not necessarily a LevelElement